Add onelogin and okta SAML docs (#1108)
CloudFire-LLC/cloudfire: docs/docs/authenticate/saml/okta.mdx

---
title: Okta
sidebar_position: 1

> description: Firezone's WireGuard based remote access platform supports using Okta as a single sign-on provider using a SAML 2.0 connector.

Reviewer: ```suggestion
description: Firezone's WireGuard®-based remote access platform supports using Okta as a single sign-on provider using a SAML 2.0 connector.
```

---

:::note
This guide assumes you have completed the prerequisite steps
outlined [here](/authenticate/saml).
:::

> Configure Okta as an identity provider using a SAML 2.0
> connector so users can authenticate with their Okta credentials.
> Follow the steps below to activate the integration.

Reviewer: If we want to maintain the same tone as the OIDC guides:

```suggestion
Firezone supports Single Sign-On (SSO) using Okta through the generic SAML 2.0 connector. This guide will walk you through how to configure the integration.
```

| Application username           | `Email`                                                                                                               |
| Update application username on | `Create and update`                                                                                                   |

[Okta's documentation](https://help.okta.com/oie/en-us/Content/Topics/Apps/Apps_App_Integration_Wizard_SAML.htm)
contains additional details on the purpose of each configuration setting. In all
the fields above, replace `firezone.company.com` with your deployment's external URL.

## Add SAML identity provider to Firezone

In the Firezone portal, add a SAML identity provider under the Security tab
by filling out the following information:

| Setting   | Value      | Notes                                                                                                                                 |
|---        |---         |---                                                                                                                                    |

> | Config ID | `Okta`     | Firezone uses this value to construct endpoints required in the SAML authentication flow (e.g., receiving assertions, login requests) |

Reviewer: ```suggestion
| Config ID | `Okta`     | Firezone uses this value to construct endpoints required in the SAML authentication flow (e.g., receiving assertions, login requests).|
```
